gulp.config.js

```diff
@@ -0,0 +1,27 @@
+/* global module, __dirname */
+var APPLICATION_DIRECTORY = 'app/src',
+  CONNECTION_PORT = 8888;
+
+var config = {
```

Comment:
Interesting. Why did you decide to move out the config? Doesn't it make the code just a teeny bit harder to read/maintain, since you now need both files open to make some changes?